tests/customerSite/menu/addCoffee/coffeeCupAddedToTotal.spec.js

```
@@ -1,22 +1,36 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
```

This file imports a module-level Allure instance: `import { allure } from 'allure-playwright';`. The implementation requirement says: **"The labels must be attached in a way that is tied to the current test via the Playwright Allure adapter API so they appear in the report."** and the corrections require: **"HIGH: Remove any fallback from testInfo.allure to a module-level singleton for attaching Allure metadata... All metadata operations must use the Playwright adapter bound to the current test."**

Using the module-level `allure` here causes labels to be attached outside the Playwright-bound test adapter and can mask misconfiguration. Please remove this import and switch to the Playwright-bound adapter (for example `const ti = test.info(); ti.allure.*`) or expose the per-test `allure` via your fixtures. Also ensure you fail loudly if the per-test adapter is not present.